core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -176,9 +178,23 @@ export class LoggedInUser extends BaseUser {
    * Navigates to the learner dashboard.
    */
   async navigateToLearnerDashboard(): Promise<void> {
+    // Reloading the page to ensure the latest state is reflected.
+    // await this.page.reload();
```

Comment:
Why is this commented out? Should drop entirely.

Feel free to clarify the name if needed though (e.g. navigateToLearnerDashboardByTypingUrl)

Reply:
Done!